tests/integration/test_users.py

```diff
@@ -154,3 +154,18 @@ def test_subscriptions(self):
         assert len(repos) > 0
         for repository in repos:
             assert isinstance(repository, github3.repos.ShortRepository)
+
+
+class TestAuthenticatedUser(IntegrationHelper):
+    """Integration tests for the AuthenticatedUser object."""
+
+    def test_repository_invitations(self):
```

Comment:
You won't need to re-record this test to move the method to the `GitHub` object. You can just rename the cassette to `GitHub_repository_invitations.json` in the cassettes directory and it should Just Work:tm: